src/keri/db/basing.py

```diff
@@ -3277,10 +3277,10 @@ def __init__(self, baser, **kwa):
         super(BaserDoer, self).__init__(**kwa)
         self.baser = baser
 
-    def enter(self):
+    def enter(self, *, temp=False):
         """"""
         if not self.baser.opened:
-            self.baser.reopen()
+            self.baser.reopen()  # reopen(temp=temp)
 
```

Comment:
Comment?

Reply:
I was mirroring Sam's comments in his PR. The comment is now removed.